src/main/java/SaveFile.java

```diff
@@ -0,0 +1,93 @@
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.stream.StreamSupport;
+
+public class SaveFile {
+
+    protected String path;
+
+    public SaveFile(String path){
+        this.path = path;
+    }
+
+    public TaskList read() throws IOException{
+
+        File t = new File(this.path);
```

Comment:
This file variable name may be too arbitrary. You have `t` for file object but a case "T" as well. You may wish to consider more meaningful variable names.